etcddb/etcddb_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"github.com/singnet/snet-daemon/blockchain"
 	"github.com/singnet/snet-daemon/escrow"
```

Comment:
Looks like `escrow` dependency here is kept only because of `func (suite *EtcdTestSuite) TestExecuteTransaction() {
` test. Is it possible to move these test under the `escrow` package?

Reply:
fixed

Reply:
@anandrgitnirman , I am very sorry if I was unclear, but I meant only `TestExecuteTransaction` test. It looked like it is only test which depends on `escrow` package and it can be moved into `prepaid_service_test.go` file. But I looked carefully on the test and I see it actually tests `EtcdClient.ExecuteTransaction` method and uses `PrepaidStorage` only as a class which uses transactions. So I think it is better to keep tests in `etcddb` and proper way is to add two different tests: one for `EtcdClient.ExecuteTransaction` and another for `PrepaidStorage`. Or make `TestExecuteTransaction` kind of integration test (which we don't have yet).

Would you mind removing last commit I think for now it is more logical.

Reply:
@vsbogd , makes sense , addressed this

Reply:
#531